//现在改为python3
echo alias python=python3 >> ~/.bashrc

source ~/.bashrc
```
版本修改为python3 😍️

Ubuntu下普通用户用sudo执行命令时报"xxx is not in the sudoers file.This incident will be reported"错误，解决方法就是在/etc/sudoers文件里给该用户添加权限。如下：

1.切换到root用户下

2./etc/sudoers文件默认是只读的，对root来说也是，因此需先添加sudoers文件的写权限,命令是:
chmod u+w /etc/sudoers

3.编辑sudoers文件
vi /etc/sudoers
找到这行 root ALL=(ALL) ALL,在他下面添加xxx ALL=(ALL) ALL (这里的xxx是你的用户名)


[global]
index-url = https://pypi.douban.com/simple
trusted-host = pypi.douban.com


echo alias pip=pip3 >> ~/.bashrc

source ~/.bashrc
安装虚拟环境
pip install virtualenv
pip install virtualenvwrapper


export WORKON_HOME=$HOME/.virtualenvs
export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
export VIRTUALENVWRAPPER_VIRTUALENV=~/.local/bin/virtualenv

source ~/.local/bin/virtualenvwrapper.sh


sudo apt install mongodb


E: Sub-process /usr/bin/dpkg returned an error code (1)

第一种方法：
sudo mv /var/lib/dpkg/info /var/lib/dpkg/info.bak //现将info目录更名

sudo mkdir /var/lib/dpkg/info //再新建一个新的info目录

sudo apt-get update


sudo apt install redis-server

```